unit-tests/src/test/scala/java/util/MapTest.scala

```diff
@@ -246,28 +243,32 @@ trait MapTest {
     val mp = factory.empty[String, String]
 
     val m = mu.Map[String, String]("X" -> "y")
-    mp.putAll(mutableMapAsJavaMap(m))
+    mp.putAll(m.toJavaMap[String, String])
     assertEquals(1, mp.size())
     assertEquals("y", mp.get("X"))
 
     val nullMap = mu.Map[String, String]((null: String) -> "y", "X" -> "y")
 
     if (factory.allowsNullKeys) {
-      mp.putAll(mutableMapAsJavaMap(nullMap))
+      mp.putAll(nullMap.toJavaMap[String, String])
       assertEquals("y", mp.get(null))
       assertEquals("y", mp.get("X"))
     } else {
       expectThrows(classOf[NullPointerException],
-                   mp.putAll(mutableMapAsJavaMap(nullMap)))
+                   mp.putAll(nullMap.toJavaMap[String, String]))
     }
   }
 
   class SimpleQueryableMap[K, V](inner: mu.HashMap[K, V])
       extends ju.AbstractMap[K, V] {
     def entrySet(): java.util.Set[java.util.Map.Entry[K, V]] = {
-      setAsJavaSet(inner.map {
-        case (k, v) => new ju.AbstractMap.SimpleImmutableEntry(k, v)
-      }.toSet)
+      inner
+        .map {
+          case (k, v) => new ju.AbstractMap.SimpleImmutableEntry(k, v)
```

Comment:
You can probably avoid the cast below with an explicit ascription here:
```suggestion
          case (k, v) =>
            new ju.AbstractMap.SimpleImmutableEntry(k, v): java.util.Map.Entry[K, V]
```